processing_r/processing/algorithm.py

```diff
@@ -217,8 +217,11 @@ def process_metadata_line(self, line):
             self.addParameter(QgsProcessingParameterFileDestination(RAlgorithm.RPLOTS, self.tr('R Plots'),
                                                                     self.tr('HTML files (*.html)'), optional=True))
             return
-        if line.lower().strip().startswith('dontuserasterpackage'):
-            self.use_raster_package = False
+        if line.lower().strip().startswith('load_raster_using_rgdal'):
```

Comment:
Here we'll also need to check for the older string too - just to avoid breaking any existing scripts

Reply:
I fixed it by including both the original keyword and new keyword.  For all cases.